tests/operators/models/test_models.py

```diff
@@ -0,0 +1,182 @@
+import pytest
+import torch
+
+from mrpro.operators.models import InversionRecovery
+from mrpro.operators.models import Molli
+from mrpro.operators.models import SaturationRecovery
+from tests import RandomGenerator
+
+
+def test_saturation_recovery_t_0():
```

Comment:
Would be great if we could parametrize these tests, such that we say something like:

```
@pytest.mark.parametrize(
    'ti, result',
    [
        # short ti
        ( 0, '0')
        # long ti
        ( 20, 'm0')
    ]
```